scipy/stats/__init__.py

```diff
@@ -265,76 +266,159 @@
    binned_statistic_2d  -- Compute a 2-D binned statistic for a set of data.
    binned_statistic_dd  -- Compute a d-D binned statistic for a set of data.
 
-Correlation functions
-=====================
+Hypothesis Tests and related functions
+======================================
+SciPy has many functions for performing hypothesis tests that return a
+test statistic and a p-value, and several of them return confidence intervals
+and/or other related information.
+
+The headings below are based on common uses of the test. Due to the variety of
+tests, any attempt at coarse-grained categorization will be imperfect. Also,
+note that tests within the same heading are not interchangeable in general
+(e.g. many have different distributional assumptions).
+
+One Sample Tests / Paired Sample Tests
+--------------------------------------
+One sample tests are typically used to assess whether a single sample was
+drawn from a specified distribution or a distribution with specified properties
+(e.g. zero mean).
 
 .. autosummary::
    :toctree: generated/
 
-   f_oneway
-   alexandergovern
+   ttest_1samp
+   binomtest
+   skewtest
+   kurtosistest
+   normaltest
+   jarque_bera
+   shapiro
+   anderson
+   cramervonmises
+   ks_1samp
+   goodness_of_fit
+   chisquare
+   power_divergence
+
+Paired sample tests are often used to assess whether two samples were drawn
+from the same distribution; they differ from the independent sample tests below
+in that each observation in one sample is treated as paired with a
+closely-related observation in the other sample (e.g. environmental factors
+are controlled between observations within a pair but not between different
+pairs). They can also be interpreted as one-sample tests (e.g.
+a test of the mean of differences between paired observations).
+
+.. autosummary::
+   :toctree: generated/
+
+   ttest_rel
+   wilcoxon
+
+Association/Correlation Tests
+-----------------------------
+
+These tests are often used to assess whether there is a relationship (e.g.
```

Comment:
There is probably a better way to write this that is more respectful of the difference between "association" and "correlation"?